crates/runtime/src/tools/mcp/server.rs

```diff
@@ -45,6 +45,18 @@ impl From<&Arc<Runtime>> for RuntimeServer {
 }
 
 impl ServerHandler for RuntimeServer {
+    fn get_info(&self) -> ServerInfo {
+        ServerInfo {
+            instructions: None,
+            capabilities: ServerCapabilities::builder().enable_tools().build(),
+            server_info: Implementation {
+                name: "Spice.ai".to_string(),
```

Comment:
I think product name is more suitable than company name here

```suggestion
                name: "Spice.ai OSS".to_string(),
```